eventkit_cloud/api/views.py

```diff
@@ -197,11 +197,22 @@ def create(self, request, *args, **kwargs):
         serializer = self.get_serializer(data=request.data)
         if (serializer.is_valid()):
             """Get the required data from the validated request."""
-            provider_tasks = request.data.get('provider_tasks')
+
+            export_providers = request.data.get('export_providers', [])
+            provider_tasks = request.data.get('provider_tasks', [])
             tags = request.data.get('tags')
             preset = request.data.get('preset')
             translation = request.data.get('translation')
             transform = request.data.get('transform')
+            if len(export_providers):
```

Comment:
This (and to be fair the check for 'provider_tasks' which you didn't write below it) is probably more readable as just 
```
if export_providers:
```
this might not be true if `None` was allowed and you wanted to throw an exception but since you pass in an empty array they are essentially the same.